README.md

```diff
@@ -43,7 +43,14 @@ Behaviors included in the model include:
  * Virus checking of original files
  * Full-text extraction from original files
 
-Check out the [Hydra::Derivatives README](https://github.com/samvera/hydra-derivatives#dependencies) for additional dependencies.
+## Dependencies
+
+Check out the [Hydra::Derivatives README](https://github.com/samvera/hydra-derivatives#dependencies) for dependencies.
+
+## Additional dependencies required for specs
+
+* ClamAV
+  * [Mac installation](https://gist.github.com/gagarine/9168c1b7e4b5f55cb3254582e30d808e)
```

Comment:
Can we just add the directions to the README instead of linking to a gist?

```
$ brew install clamav
$ cp /usr/local/etc/clamav/freshclam.conf.sample /usr/local/etc/clamav/freshclam.conf
$ freshclam
```

Reply:
Updated with that change.